.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,188 @@
+name: Deploy binaries and create release
+on:
+  release:
+    types:
+      - published
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  github_build:
+    name: Build release binaries
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-linux-gnu
+
+          - target: x86_64-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-linux-musl
+
+          - target: i686-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-i686-unknown-linux-musl
+
+          - target: aarch64-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-aarch64-unknown-linux-musl
+
+          - target: arm-unknown-linux-musleabi
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-arm-unknown-linux-musleabi
+
+          # - target: armv7-unknown-linux-musleabi
+          #   os: ubuntu-latest
+          #   name: prometheus_wireguard_exporter-armv7-unknown-linux-musleabi
```

Comment:
Why is this one commented? 🤔 

It's quite an important one, I think most arm devices are armv7 today

Reply:
I couldn't make this work. Any suggestion of someone more experienced with Rust and Cargo is welcome.

Reply:
🤔 I had the same issue when adding cross compiled Rust binaries for the Docker image and solved it somehow.. Now I don't remember how even when looking at my Dockerfile 😢

What's the error log you get? It's also rather strange you get the error for armv7 but not arm for example.

Regarding the apple-aarch64 I'm not sure since I'm not building it for Docker, so maybe it's another issue. But the Dockerfile is definitely building the binary for `armv7-unknown-linux-musleabi` so we should be able to solve it.